--- origsrc/libpng-1.4.14/configure.ac	2014-11-20 09:16:22.000000000 -0600
+++ src/libpng-1.4.14/configure.ac	2014-12-31 04:27:00.650946700 -0600
@@ -67,17 +67,6 @@ esac
 LIBPNG_DEFINES=$LIBPNG_DEFINES
 AC_SUBST(LIBPNG_DEFINES)
 
-AC_MSG_CHECKING([if libraries can be versioned])
-# Special case for PE/COFF platforms: ld reports
-# support for version-script, but doesn't actually
-# DO anything with it.
-case $host in
-*cygwin* | *mingw32* )
-    have_ld_version_script=no
-    AC_MSG_RESULT(no)
-;;
-* )
-
 AC_MSG_CHECKING([if using Solaris linker])
 SLD=`$LD --version 2>&1 | grep Solaris`
 if test "$SLD"; then
@@ -89,6 +78,17 @@ else
 fi
 AM_CONDITIONAL(HAVE_SOLARIS_LD, test "$have_solaris_ld" = "yes")
 
+AC_MSG_CHECKING([if libraries can be versioned])
+# Special case for PE/COFF platforms: ld reports
+# support for version-script, but doesn't actually
+# DO anything with it.
+case $host in
+*cygwin* | *mingw32* )
+    have_ld_version_script=no
+    AC_MSG_RESULT(no)
+;;
+* )
+
 if test "$have_solaris_ld" = "yes"; then
     GLD=`$LD --help < /dev/null 2>&1 | grep 'M mapfile'`
 else
